### PR TITLE
Adds more logging to aid debugging timeout issue

### PR DESCRIPTION
**Pull request by tlunter**
@bfulton @nahiluhmot

Depends on: https://github.com/swipely/dockly-util/pull/7

Gives more information regarding processes and time

### CODE REVIEW

File: lib/aerosol/runner.rb [hunk 1]
```diff
@@ -72,8 +72,13 @@ def wait_for_new_instances
         info "waiting for instances to be live (#{remaining_instances.count} remaining)"
         debug "current instances: #{current_instances.map(&:id)}"
         debug "live instances: #{live_instances.map(&:id)}"
-        live_instances.concat(remaining_instances.select { |instance| healthy?(instance) })
+        new_live_instances = remaining_instances.select { |instance|
+          debug "Checking if #{instance.id} is healthy"
```
Comment (bfulton) [inline lib/aerosol/runner.rb]: You could leave this block alone if you moved the debug log to the first statement in `healthy?`

**Comment by bfulton** [comment]
You had also mentioned that there was character-based (instead of line-based) reading from ssh that's making output hard to read. Can that be fixed here too?

**Comment by tlunter** [comment]
@bfulton updated

**Comment by bfulton** [comment]
+1

**Pull request merged**